### PR TITLE
Added a new question on git integrity in git-quiz.ts

### PR DESCRIPTION
**Pull request by deedeecee**
Checklist:

- [ x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [ x] My pull request has a [descriptive title](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request?id=prepare-a-good-pr-title) (**not** a vague title like `Update index.md`)

Closes #XXXXX

### CODE REVIEW

File: src/data/git-quiz.ts [hunk 1]
```diff
@@ -430,6 +430,16 @@ const gitQuiz = [
       "The git push -f command will force a push request. This is usually fine for pull request branches because nobody else should have cloned them.",
     Link: "https://www.freecodecamp.org/news/git-cheat-sheet/",
   },
+  {
+    Question: "Why is it impossible to make changes without Git knowing about it?"
```
Comment (chefgs) [inline src/data/git-quiz.ts:434]: @deedeecee 
Add `,` comma at the end of line to resolve the linting issues.

Comment (deedeecee) [reply src/data/git-quiz.ts:434]: Thanks for the comment, sir. 
I'll make the necessary changes as you asked.

Comment (chefgs) [reply src/data/git-quiz.ts:434]: @deedeecee Still the comma has not been added at the end of line no 434. Please read PR comments properly and make the changes.
I believe your PR will be approved, only If the PR does not have any linting issues. So fix the error in line no 434.

**Review by jdwilkin4 — CHANGES_REQUESTED** [review]
Thank you for your contribution!

Please review the requested changes.
Once those changes have been implemented, then we can merge your PR into main. 👍

**Review by jdwilkin4 — APPROVED** [review]
Congrats on your first contribution to our repository. 🎉
We look forward to future contributions. 👍

**Pull request merged**